.github/workflows/container-ci.yaml

```diff
@@ -45,17 +45,26 @@ on:
         default: false
         required: true
         type: boolean
+      ref:
+        description: 'Enter Git Ref:'
+        required: true
+        default: 'main'
+        type: string
```

Comment:
I removed this input because it's redundant.

Reply:
I have removed this!

Reply:
It looks like the input is still present, just not required anymore which can lead to a failure.